pkgs/servers/sunshine/default.nix

```diff
@@ -84,10 +84,8 @@ stdenv'.mkDerivation rec {
   nativeBuildInputs = [
     cmake
     pkg-config
-    autoPatchelfHook
```

Comment:
It's there's no comment explaining what exactly autopatchelf's responsibility here is

Reply:
Either way, it's weird that autoPatchelf is there, but we shouldn't be spending time on removing it in this PR because that's out of scope

Reply:
@SomeoneSerge upstream vendors patched FFmpeg binaries and I kinda got tired of fighting their packaging to patch them out, so `autoPatchelf` let me just stop worrying and let it work. Agree I should have added some verbiage explaining it at some point -- that's a miss on my part.